docker-compose.yml

```diff
@@ -34,6 +34,7 @@ web:
     - APP_CACHE_REDIS_URL=redis://redis:6379/0
     - APP_ACCOUNTS_SESSION_REDIS_URL=redis://redis:6379/1
     - APP_BROKER_URL=amqp://guest:guest@rabbitmq:5672/
+    - APP_CELERY_BROKER_URL=amqp://guest:guest@rabbitmq:5672/ # Celery 4
```

Comment:
Maybe add also a `# Celery 3` comment to the line above, or simply remove both comments since this will be "documented" in upstream Invenio, as it were?